src/Units/UnitState.java

```diff
@@ -162,7 +162,7 @@ public int alterHP(int change, boolean allowOver)
 
     health = Math.max(1, health + healthFromHP(realChange));
     // Round HP up, if healing
-    if( change > 0 )
+    if( change >= 0 )
       health = healthFromHP(getHP());
```

Comment:
OK, I'm curious. What significant effect does this change have?

Reply:
I wanted to use `alterHP(0)` to round up the health to the maximum for that HP value.
I could write a new function instead, but we already have too many health functions.